jib-maven-plugin/src/integration-test/java/com/google/cloud/tools/jib/maven/BuildImageMojoIntegrationTest.java

```diff
@@ -52,13 +52,9 @@
 public class BuildImageMojoIntegrationTest {
 
   @ClassRule
-  public static final LocalRegistry localRegistry1 =
+  public static final LocalRegistry localRegistry =
```

Comment:
It seems like if we only use one registry, then `buildAndRunComplex` doesn't need to accept registry info as parameters?

Reply:
Ohhh, that's right. Also no need to accept username and password info.